lib/Validator/Constraints/FieldValueValidatorMessages.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\RepositoryForms\Validator\Constraints;
+
+use JMS\TranslationBundle\Translation\TranslationContainerInterface;
+use JMS\TranslationBundle\Model\Message;
+
+class FieldValueValidatorMessages implements TranslationContainerInterface
```

Comment:
Please add PhpDoc stating why you need this class so noob like me can understand that as well ;)
I mean, it's used for JMS Translation extractor, right?

Reply:
Done